crio/datadog_checks/crio/__about__.py

```diff
@@ -2,4 +2,4 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
-__version__ = '1.0.0'
+__version__ = '0.0.1'
```

Comment:
Why change the version here?

Reply:
Release command needs to bump the version

Reply:
Why not 1.0.0 though?

Reply:
Ahh ok. Makes sense. Thanks